packages/firebase_dynamic_links/firebase_dynamic_links_platform_interface/lib/src/platform_interface/platform_interface_firebase_dynamic_links.dart

```diff
@@ -0,0 +1,91 @@
+// ignore_for_file: require_trailing_commas
+// Copyright 2021, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:firebase_core/firebase_core.dart';
+import 'package:firebase_dynamic_links_platform_interface/src/platform_interface/platform_interface_dynamic_link_builder.dart';
+import 'package:meta/meta.dart';
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+
+import '../../firebase_dynamic_links_platform_interface.dart';
```

Comment:
This can just be the full import `import 'package:firebase_dynamic_links_platform_interface/firebase_dynamic_links_platform_interface.dart';